caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters for the entire run of initialization and OnACID
+
+            estimates: Estimates, optional
+                Estimates object to load an existing model
+
+            path: str, optional
+                path to a saved OnACID model on disk
```

Comment:
should we mention what format the model is in? Or point at a doc maybe for how to make on of these?

Reply:
Pat good point., We should probably have an example of this in use in a demo notebook at some poin, given that we want to move toward online as the next gen.

Reply:
I assuming it's the hdf5 files? Anyways I think the better way to do this (and the estimates above) is to have a `@classmethod`. Such as `OnACID.from_estimates()`, `OnACID.from_hdf()`, etc.